python/ray/util/client/__init__.py

```diff
@@ -70,9 +70,15 @@ def connect(self,
             conn_info = self.client_worker.connection_info()
             self._check_versions(conn_info, ignore_version)
             return conn_info
-        except Exception:
+        except Exception as e:
```

Comment:
What's the motivation for this change? I think a lot of users will be confused about how to set the log level and it will just hinder debugging.

Reply:
I can revert; it's just an uglier stacktrace.